test/unit/testCheckSummaryPDF.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+
+const requireDir = require('require-directory');
+const TestWrapper = require('test/util/TestWrapper');
+const TaskList = require('app/steps/ui/tasklist/index');
+const translations = requireDir(module, '../../app/resources/en/translation');
+const {assert} = require('chai');
+const he = require('he');
+
+describe('checkAnswerPDF', () => {
+    let testWrapper;
+    let sessionData = require('test/data/checkQuestionAnswerPDF');
+    const headingMedium = 'heading-medium';
+    const headingSmall = 'heading-small';
+
+    beforeEach(() => {
+        testWrapper = new TestWrapper('Summary');
+    });
+
+    afterEach(() => {
+        testWrapper.destroy();
+    });
+
+    describe('Verify Check Answers Summary Object is built', () => {
+
+        it('test check answers summary object populated', (done) => {
```

Comment:
This test should be a component test rather than a unit test and we have some component methods that we could perhaps use for checking the content.

Reply:
moved to component test